helpers/bumper.py

```diff
@@ -28,7 +28,6 @@
 file_patterns = [
     "*/examples/*/*.y*ml",
     "helpers/examples.mk",
-    "*/README.md",
```

Comment:
Why are the README excluded?

We'll need to bump the versions in README to 7.7.0 before the release.

Reply:
Good catch I pushed c54c8c5